docs/rawdata/raw_data_upload.md

```diff
@@ -63,53 +63,76 @@ For every registration task, the data needs to reside in a folder with the follo
 
 ```text
 |- my-registration-batch  // folder name is irrelevant
- |- file1_1.fastq.gz
- |- file1_2.fastq.gz
- |- file2_1.fastq.gz
- |- file2_2.fastq.gz
  |- metadata.txt  // mandatory!
+ |- file1_1.fastq.gz // all files except for metadata.txt serve as examples
+ |- file1_2.fastq
+ |- report.pdf
+ |- summary.html
+```
+
+
+
+!!! warning
+    Ensure that the uploaded folder name and files do not have a whitespace within their name
+
+You can upload folders in the same way. Everything at the top level of your created folder is
```

Comment:
Why no !!!info?

Reply:
I removed the info since it's now an entire section with examples for uploading a folder